### PR TITLE
Set overwrite=true for various fields.

### PR DESCRIPTION
**Pull request by simitt**
This is required since the `fields.yml` files are now duplicated per event type because of the Elastic agent package restructure,
The index pattern generator requires to explicitly set overwrite:true for
duplicate fields.

### CODE REVIEW

File: docs/fields.asciidoc [hunk 1]
```diff
@@ -43,6 +43,8 @@ Processor name.
 
 type: keyword
 
+{yes-icon} {ecs-ref}[ECS] field.
```
Comment (jalvz) [inline docs/fields.asciidoc:46]: The logic for discerning ECS fields in the docs is wrong, would you mind create an issue for that when you get a change?

**Review by jalvz — APPROVED** [review]
As a follow up we could add a test to ensure all fields have `overwrite:true`

**Comment by simitt** [comment]
> As a follow up we could add a test to ensure all fields have overwrite:true

If they aren't the `areKibanaObjectsUpdated` test step actually fails - we just ignored it.

File: docs/fields.asciidoc (hunk 1 above)
Comment (simitt) [reply docs/fields.asciidoc:46]: great catch!
https://github.com/elastic/apm-server/issues/4554

**Pull request merged**